lib/dal/src/func/binding.rs

```diff
@@ -351,6 +355,51 @@ impl FuncBinding {
         let (context, rx) = FuncDispatchContext::new(ctx);
         Ok((func, execution, context, rx))
     }
+
+    pub async fn execute_dummy(
+        ctx: &DalContext,
+        func_id: FuncId,
+        func_backend_kind: FuncBackendKind,
+        args: serde_json::Value,
+        execution_key: String,
+        component_id: ComponentId,
+    ) -> FuncBindingResult<(serde_json::Value, Vec<OutputStream>)> {
+        let before = before_funcs_for_component(ctx, component_id)
+            .await
+            .map_err(Box::new)?;
+
+        let func_binding = Self::new(ctx, args, func_id, func_backend_kind).await?;
+
+        let (func, _execution, context, mut rx) = func_binding.prepare_execution(ctx).await?;
+
+        // TODO(nick): if we could do this without a rollback that would be great, especially since
+        // only have to work with the graph in memory.
+        ctx.rollback().await?;
```

Comment:
We likely do not need this in the future. Just a port of the original code that works because `FuncBinding` stuff still uses the old tables.